# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1322039650 0
#      Wed Nov 23 09:14:10 2011 +0000
# Node ID 1f6d17b383f80b929cdc6b7536ea8c7e8752a1fd
# Parent  ec435c4d81989bd8818fddd073ee5454dbc27fdd
Add notes about new handle graphics functions.

* NEWS: Add notes about added handle graphics functions.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -100,16 +100,29 @@ Summary of important user-visible change
     "fortran-indexing"    : removed
     "precedence-change"   : removed
     "string-concat"       : renamed to "mixed-string-concat"
 
  ** Matlab-compatible preference functions added:
 
     addpref  getpref  ispref  rmpref  setpref
 
+ ** Matlab-compatible handle graphics functions added:
+
+    guidata         uipanel        uitoolbar
+    guihandles      uipushtool     uiwait
+    uicontextmenu   uiresume       waitfor
+    uicontrol       uitoggletool
+
+    Except uiwait and uiresume, the uiXXX functions are not supported on
+    the FLTK backend.
+
+    The gnuplot backend does not support any of the uiXXX functions as well
+    as the waitfor function.
+
  ** New keyword parfor
     parfor (parallel for loop) is now recognized as a valid keyword.
     Implementation, however, is still mapped to an ordinary for loop.
 
  ** Other new functions added in 3.6.0:
 
     is_dq_string    nthargout    usejava     
     is_sq_string    python       waitbar
